### DISCUSSION TITLE
Can we have a button to generate a new LXD Client Certificate file and download it?

### DISCUSSION THREAD

**Discussion by robmlee** [lxdware/lxd-dashboard#19] [Ideas] [answered]
Hi there, I am using lxd-dashboard 2.3.0. It's really a awesome Web GUI tool for LXD. I am running two Labs, one for work at office, and one for myself at home.  After I setup a lxd-dashboard container at office lab, I exported it and imported it at home lab. And I found that there's no way to have a new LXD Client Certificate. So I just wonder can lxd-dashboard add a button that can create a new LXD Client Certificate token file and followed by a automatic download process?

**Comment by matthewalanpenning** [comment, marked as answer]
Hello robmlee,
Thanks for the wonderful feedback. That is a great suggestion. I do have the ability to delete the client LXD certificate built into my next release which I am working to get out by the end of November. When deleting the existing client certificate a new one will be automatically generated. I wrote it to replace expired client certificates, but it should work for your use case as well.

**Reply by robmlee** [reply]
Thanks Matthewalanpenning! I'm so glad to hear that and looking forward to the new release.
